src/parser.jl

```diff
@@ -130,7 +130,12 @@ end
 curline(ts::TokenStream)  = ts.lineno
 filename(ts::TokenStream) = symbol("none")
 
-line_number_node(ts) = LineNumberNode(curline(ts))
+if VERSION < v"0.4-dev+7034"
+    line_number_node(f, n) = LineNumberNode(n)
```

Comment:
You are right only `line_number_node(ts) = LineNumberNode(symbol(""), curline(ts))` is needed. And indeed the Package is loadable but not able to parse anything, what this PR does (I will update of course) is make `JuliaParser.Parser.parse` work again also, I think I choose bad names for the PR.